flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClient.java

```diff
@@ -102,6 +109,14 @@ public Fabric8FlinkKubeClient(
         this.maxRetryAttempts =
                 flinkConfig.getInteger(
                         KubernetesConfigOptions.KUBERNETES_TRANSACTIONAL_OPERATION_MAX_RETRIES);
+        this.initialRetryInterval =
+                flinkConfig.getOptional(
+                                KubernetesConfigOptions.KUBERNETES_TRANSACTIONAL_OPERATION_INITIAL_RETRY_DEALY)
+                        .orElse(KubernetesConfigOptions.KUBERNETES_TRANSACTIONAL_OPERATION_INITIAL_RETRY_DEALY.defaultValue());
+        this.maxRetryInterval =
+                flinkConfig.getOptional(
+                                KubernetesConfigOptions.KUBERNETES_TRANSACTIONAL_OPERATION_MAX_RETRY_DEALY)
+                        .orElse(KubernetesConfigOptions.KUBERNETES_TRANSACTIONAL_OPERATION_MAX_RETRY_DEALY.defaultValue());
```

Comment:
There's no need to use `getOptional().orElse()`. A simple `get()` should automatically apply the default value if not configured.

```suggestion
        this.initialRetryInterval =
                flinkConfig.get(
                                KubernetesConfigOptions.KUBERNETES_TRANSACTIONAL_OPERATION_INITIAL_RETRY_DEALY);
        this.maxRetryInterval =
                flinkConfig.get(
                                KubernetesConfigOptions.KUBERNETES_TRANSACTIONAL_OPERATION_MAX_RETRY_DEALY);
```